src/backend/distributed/worker/worker_drop_protocol.c

```diff
@@ -57,6 +59,13 @@ worker_drop_distributed_table(PG_FUNCTION_ARGS)
 	CheckCitusVersion(ERROR);
 	EnsureSuperUser();
 
+	if (!OidIsValid(relationId))
+	{
+		ereport(NOTICE, (errmsg("relation %s does not exist, skipping",
```

Comment:
note to reviewer : is NOTICE is good here ? That's what postgres does to inform when cascading etc.

Reply:
@onderkalaci any comments on `NOTICE` ?